### ISSUE TITLE
Help needs for the running code

### ISSUE THREAD

**Issue by sinhae817** [xiaomengyc/I2C#2]
Hello.
I'd recently read your paper and want to try the code to run and see the result of it and understand the code.
But, for the some of the reasons, I had the trouble to run your code, and couldn't understand the structure of those.
1. In train  or val function, the return parameters/data from the dataloader and initialized name couldn't match for me, so, I couldn't understand how the data come out
2. couldn't understand which parts of the code  are working for the SC and GC
3. result image file save issue - no saving
4. validation accuracy issue
5. validation saving issue 
... and some mores
even to  come out to run the validation and training I tried  to edit little,  but, I really want to see the your  architectures' work  and understand.
If you didn't update your code, could you? or can you help  me to at least the 
train_frame.py train function with dataloader.

Thanks,

**Comment by xiaomengyc** [comment]
Hi @sinhae817,
Sorry for the late reply.
You were seen running into a lot of trouble, and I am very sorry to hear that. I can hardly understand what your real problem is. I hope you can describe it more and specifically. Or, you can drop me an email to discuss detailed understandings of this project.
